### PR TITLE
Remove duplicate redundant words

### PR DESCRIPTION
**Pull request by grimreaper**

### CODE REVIEW

**Comment by thpr** [comment]
Code changes look good to me, have not reviewed data

File: data/pathfinder/dreamscarred_press/path_of_war/pow_maneuvers.lst [hunk 1]
```diff
@@ -926,7 +926,7 @@ Iron Defender's Riposte.MOD				DESC:The Iron Tortoise master is wary of the dang
...
+Iron Hide Stance.MOD					DESC:Standing strong in theater of war, the Primal Fury master stands as a pillar of blood-soaked iron amidst the bodies of those too weak to live in his presence. While in this stance, the initiator's Strength bonus to damage is doubled (or multiplied to 2.5 if using a weapon two- handed), and the initiator is treated as one size category larger (so if the initiator is Medium-sized, then he counts as Large-sized) when it benefits his CMB and CMD (such as resisting trip attacks or overrun attempts). Additionally, the initiator gains a damage reduction of 5/-.|PRERULE:1,DisplayFullSpell
```
Comment (LegacyKing) [inline data/pathfinder/dreamscarred_press/path_of_war/pow_maneuvers.lst]: Can we verify this is accurate? Not a duplicate removal here.

File: data/pathfinder/dreamscarred_press/path_of_war/pow_maneuvers.lst [hunk 2]
```diff
@@ -940,7 +940,7 @@ Lion Lord's Agony.MOD					DESC:With a fury born of desperation and determination
...
-Meteoric Collision.MOD					DESC:The disciple makes a charge at an opponent within the theater of war with the power and force of a falling meteor, striking hard and bowling his opponent over. The initiator makes a bull rush attempt as part of a charge, and if the initiator successfully bull rushes his opponent, the fury of his strike inflicts 6d6 points of damage to his target as he bull rushes him. This bull rush attempt does not provoke attacks of opportunity.|PRERULE:1,DisplayFullSpell
+Meteoric Collision.MOD					DESC:The disciple makes a charge at an opponent within theater of war with the power and force of a falling meteor, striking hard and bowling his opponent over. The initiator makes a bull rush attempt as part of a charge, and if the initiator successfully bull rushes his opponent, the fury of his strike inflicts 6d6 points of damage to his target as he bull rushes him. This bull rush attempt does not provoke attacks of opportunity.|PRERULE:1,DisplayFullSpell
```
Comment (LegacyKing) [inline data/pathfinder/dreamscarred_press/path_of_war/pow_maneuvers.lst]: Ditto. Not a duplicate.

File: data/pathfinder/dreamscarred_press/path_of_war/pow_maneuvers.lst [hunk 3]
```diff
@@ -1002,7 +1002,7 @@ Scarlet Riposte.MOD					DESC:Within the focus of the disciple's mind, pathways t
...
+Shards of Adamantine Strike.MOD			DESC:It is said that masters of the discipline are capable of using their hand in the manner of striking serpent and punch through solid stone as if it were straw, and disciples with this maneuver are those who can. Taking this training to theater of war, a punch so potent that it can leave the foe sickened with pain is what this art form can deliver. The initiator makes an attack against the target creature and if successful, this attack inflicts an additional 12d6 points of damage that ignores the target's damage reduction or an object's hardness. If striking a living creature, the target must make a Fort save (DC 17 + primary initiator attribute modifier) or be nauseated with the pain of the strike for 1d4 rounds.|PRERULE:1,DisplayFullSpell
```
Comment (LegacyKing) [inline data/pathfinder/dreamscarred_press/path_of_war/pow_maneuvers.lst]: Ditto... looks like this file is removing "the" from in front of "theater". I would guess the search and replace method was looking for "the the" but wasn't using WHOLE word, so "the" in "theater" was caught.

**Comment by LegacyKing** [comment]
Reverse the changes to file pow_maneuvers.lst and this passes content inspection.

**Comment by LiaungYip** [comment]
The stuff in the documentation files looks all OK.

**Comment by grimreaper** [comment]
I repushed the branch with pow_maneuvers.lst fixed.

**Comment by LegacyKing** [comment]
Thanks

**Pull request merged**